### DISCUSSION TITLE
ILI9488 3.5 inch Sunlight Readable IPS on ESP32

### DISCUSSION THREAD

**Discussion by Florian-L** [Bodmer/TFT_eSPI#3115] [Compatible displays and setup files]
Hello! I have this display: https://de.aliexpress.com/item/32829704426.html
It should be the variant "IPS ILI9488 Touch".
I have wired it up according to "Setup16_ILI9488_Parallel.h". However, the display has no pin labeled "DC". I have tried the "RS" pin, as I found someone else apparently has done that successfully.
But my display remains black, even though I can see the backlight is on.
I have verified my settings with "Read_User_Setup". When I try th "TFT_ReadWrite_Test", all tests return 
  Pixel value read    = 0
  ERROR                 ^^^^
Does anyone have any ideas what I could try to make it work?
Regards,
Florian
![S49e1478f7e7843e5957aa8da65682d6c8](https://github.com/Bodmer/TFT_eSPI/assets/4710791/3778db5c-f1d7-446b-bc02-0e5af814dd9b)

**Comment by Florian-L** [comment]
Ok, I guess I need to do something with the "IM" pins? But what?

**Comment by Florian-L** [comment]
I probably have a hardware issue. Or I am just lacking proper undestanding?
I connected IM0 and IM1 to 3.3v to set the display to 8 bit bus mode, and now it displays something, but seems to crash right away. Here is a screenshot of the Colour_Test. The ESP32 is fine, it seems to be the display that is crashed.
![IMG_2812](https://github.com/Bodmer/TFT_eSPI/assets/4710791/cbbde28b-eab1-4bf7-92f2-eb3c635986a7)
<img width="467" alt="Bildschirmfoto 2024-01-19 um 11 12 18" src="https://github.com/Bodmer/TFT_eSPI/assets/4710791/0b8e45ea-8427-4b97-a06b-31ddd95de45f">

**Comment by Bodmer** [comment]
I am guessing the vendor sent you a display with a different driver. The markings look like this one on the web link above, which is has an R61529 driver:

![image](https://github.com/Bodmer/TFT_eSPI/assets/15824805/71920a61-27f2-49e3-8b94-39c713d784a7)

**Reply by Bodmer** [reply]
Incidentally, the R61529 chip is not supported by this library.

**Reply by Florian-L** [reply]
By the way, thanks for all your effort!

**Comment by Florian-L** [comment]
Ok, thanks. I guess I'm giving up on this display then.

**Comment by Bodmer** [comment]
It is almost working. It appears the initialisation is failing and not all pixel data is being recognised. Keep connections short, especially 0V (GND) and try reducing the SPI clock rate in the setup file to 10MHz.

**Comment by dadyeah** [comment]
I'm also struggling to get this display (ILI9488 sunlight readable) to work in SPI 3 wire mode,
If anyone has succeeded I would appreciate any info

**Comment by dexter323i** [comment]
@dadyeah are you still working on this project? This display inded has R61529 driver, and it can work with TFT_eSPI library, in 8 bit parallel modee. Use this PR: https://github.com/Bodmer/TFT_eSPI/pull/2715
